recipes/flake8-blind-except/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "flake8-blind-except" %}
+{% set version = "0.1.1" %}
+{% set sha256 = "aca3356633825544cec51997260fe31a8f24a1a2795ce8e81696b9916745e599" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - setuptools
```

Comment:
is setuptools a runtime dependency?

Reply:
https://github.com/elijahandrews/flake8-blind-except/blob/master/setup.py
Line 25.

Reply:
If it is not a run requirement, please let me know and I will remove it from the list of run requirement.

Reply:
It does not seem to be a direct runtime depenendcy, looking at the code https://github.com/elijahandrews/flake8-blind-except/blob/master/flake8_blind_except.py

Reply:
Thanks. I will remove setuptools from the list of run dependencies.